package.json

```diff
@@ -67,6 +72,6 @@
   "prettier": {
     "printWidth": 100,
     "trailingComma": "all",
-    "singleQuote": true
+    "singleQuote": true,
```

Comment:
does JSON allow trailing commas without freaking out?

Reply:
nope, thanks for the catch.